src/engine.py

```
@@ -3939,18 +4059,22 @@ async def _handle_caller_stasis_start_hybrid(self, caller_channel_id: str, chann
                             if meta and meta.get("context"):
                                 session.context_name = str(meta.get("context") or "").strip() or session.context_name
                                 try:
-                                    await self.ari_client.set_channel_var(caller_channel_id, "AI_CONTEXT", session.context_name or "")
+                                    await self._set_outbound_agent_channel_vars(
+                                        caller_channel_id,
+                                        session.context_name or "",
+                                    )
                                 except Exception:
                                     pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)